sound/soc/sof/intel/hda-dsp.c

```diff
@@ -614,6 +614,7 @@ static int hda_suspend(struct snd_sof_dev *sdev, bool runtime_suspend)
 #if IS_ENABLED(CONFIG_SND_SOC_SOF_HDA)
 	struct hdac_bus *bus = sof_to_bus(sdev);
 #endif
+	u32 j;
```

Comment:
this is just a counter / array index. A simple `(unsigned) int` would do

Reply:
ack